amundsen_application/static/js/interfaces/Enums.ts

```diff
@@ -0,0 +1,17 @@
+export enum SendingState {
+  ERROR = "error",
+  IDLE = "idle",
+  WAITING = "waiting",
+  COMPLETE = "complete"
+};
+
+export enum ResourceType {
+  table = "table",
+  user = "user",
+  dashboard = "dashboard",
+};
+
+export enum UpdateMethod {
+  PUT = 'PUT',
+  DELETE = 'DELETE',
+};
```

Comment:
Not sure about grouping by enums. I feel like they would be better situated inside their respective files (e.g. putting `ResourceType` in the `Resources` file.

Reply:
`UpdateMethod` is a shared enum (used for updating tags, owners, and anything else that will have to use our "batch editing" hack) and I needed a place to put it for now.

I'm not opposed to putting the other enums back within the context in which they are used though. Sound good?